src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -861,6 +861,18 @@ prep_local_assembly (RpmOstreeSysrootUpgrader *self,
   if (treespec == NULL)
     return FALSE;
 
+  RpmOstreeRefspecType refspec_type;
+  const char *refspec;
+  rpmostree_origin_classify_refspec (self->origin, &refspec_type, &refspec);
```

Comment:
I feel like at this point, we should start adding helpers like:

```c
if (rpmostree_origin_is_rojig (self->origin))
  ...
```

But we can do that later as well.

Reply:
Good idea, I'll do that in a separate PR.